tests/testthat/test-recipe.R

```diff
@@ -0,0 +1,70 @@
+skip_if_not_installed("recipes")
+skip_if_not_installed("plumber")
+
+library(plumber)
+library(recipes)
+
+trained_rec <-
+    recipe(mpg ~ disp + wt, mtcars) %>%
+    step_ns(wt) %>%
+    prep(retain = FALSE)
+
+v <- vetiver_model(trained_rec, "car-splines", prototype_data = mtcars[c("disp", "wt")])
+
+test_that("can print recipe", {
+    expect_snapshot(v)
+})
+
```

Comment:
I would normally have a test here like this:

```r
test_that("can predict recipe", {
    preds <- predict(v, mtcars)
    expect_equal(<<blah blah blah>>)
})
```

But I don't think that's possible for recipes. The `predict` method for a vetiver model does `bundle::unbundle()` and then calls `predict` on what is inside. I guess we could add a `bake` method for a vetiver model if needed? This is separate from the API where we can say exactly what to do at the endpoint.

Reply:
For more clarity, this is also separate from calling `predict()` on a remote vetiver endpoint, which would also work. What we _don't_ have a way to do right now is read the recipe back into memory from remote storage (a pin) and then call `bake()` on it, without the user manually getting out the recipe object themselves and unbundling it.

Reply:
@isabelizimm do you mind summing up here what the situation is for unsupervised models from scikit-learn as deployed by vetiver? These models typically have a `predict` method so this is not a problem in Python, right?

Reply:
Looking at just the clustering algorithms from scikit-learn, most of them have a predict method. You can use these in a Pipeline (similar to workflow), same as other models. Vetiver Python doesn't look for supervised/unsupervised models, only if it is coming from scikit-learn, so it will return the outputs of the predict method as expected.

If one of the unsupervised learning models that do NOT have a predict method are used as the last element in a Pipeline, there will be an error along the lines of `model has no predict method`.

FWIW: (clustering algorithms with predict: k-means, bisecting k-means, affinity propagation, mean shift, BIRCH, Gaussian mixture. do NOT have predict: spectral clustering, agglomerative clustering, DBSCAN, OPTIC)